packages/amplify-graphql-directives/src/directives/function.ts

```diff
@@ -1,12 +1,22 @@
 import { Directive } from './directive';
 
 const name = 'function';
+// export type FunctionInvocationType = 'RequestResponse' | 'Event'
```

Comment:
nit: remove or reinstate and make `FunctionDirectiveDefaults.invocationType` a FunctionInvocationType rather than a string. (Future PR is fine)

Reply:
I can't believe that slipped through. Thanks -- addressed in https://github.com/aws-amplify/amplify-category-api/pull/2868